app/views/components/buttons/save_and_continue_button.scala.html

```diff
@@ -0,0 +1,21 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.util.SaveAndContinue
+
+@(messageKey: String = "site.save_and_continue")(implicit messages: Messages)
+
+<button id="submit" class="button" name="@SaveAndContinue.toString">@messages(messageKey)</button>
```

Comment:
Hmm, I think we had save and contrinue button. What's this?

Reply:
Hmmm there's `submit_button` but it is subtly different as it is wrapped in a `div class="section"` and doesn't have the `name` attribute set

Reply:
I think the old submit button should be replaced as this one is more reusable.

It's not wrapped in a div so can be used in conjunction with the add button, or in any other context